src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -96,15 +96,18 @@ class AccessibleView extends Disposable {
 	}
 
 	show(provider: IAccessibleContentProvider): void {
+		let view: IDisposable | undefined;
 		const delegate: IContextViewDelegate = {
 			getAnchor: () => this._editorContainer,
 			render: (container) => {
-				return this._render(provider, container);
+				view = this._render(provider, container);
+				return view;
 			},
 			onHide: () => {
 				if (provider.options.type === AccessibleViewType.HelpMenu) {
 					this._accessiblityHelpIsShown.reset();
 				}
+				view?.dispose();
```

Comment:
Since you're returning view from render, I think the idea is that the owner of the `IContextViewDelegate` (`_contextViewService`) manages disposal. This probably means that this will often throw since `DisposableStore`s can only be disposed once.